storage/innobase/os/os0file.cc

```diff
@@ -2712,31 +2799,16 @@ bool os_file_set_sparse_win32(os_file_t file, bool is_sparse)
 		FSCTL_SET_SPARSE, &sparse_buffer, sizeof(sparse_buffer), 0, 0,&temp);
 }
 
-
-/**
-Change file size on Windows.
-
-If file is extended, the bytes between old and new EOF
-are zeros.
-
-If file is sparse, "virtual" block is added at the end of
-allocated area.
-
-If file is normal, file system allocates storage.
-
-@param[in]	pathname	file path
-@param[in]	file		file handle
-@param[in]	size		size to preserve in bytes
-@return true if success */
 bool
-os_file_change_size_win32(
+os_file_set_size(
 	const char*	pathname,
 	os_file_t	file,
 	os_offset_t	size)
 {
 	LARGE_INTEGER	length;
 
 	length.QuadPart = size;
+	ut_ad(!(size & 4095));
```

Comment:
I added the assertion because there was one in the common `os_file_set_size()` function. I agree that we can do without this one; d09426f9e60fd93296464ec9eb5f9d85566437d3 claims that the issue does not affect Microsoft Windows.